src/integrationTest/java/uk/gov/hmcts/reform/pdf/service/integration/GeneratedPDFContentV2Test.java

```diff
@@ -47,27 +64,126 @@ private static String textContentOf(byte[] pdfData) throws IOException {
     @SuppressWarnings("unchecked")
     @Test
     public void shouldCreateExpectedPdfFromPlainHtmlTemplate() throws Exception {
-        assertThatGeneratedPdfContains("<html><body>Hello!</body></html>", Collections.EMPTY_MAP, "Hello!");
+        Mockito.when(authorisationService.authorise(Mockito.anyString())).thenReturn("test-service");
```

Comment:
```java
        authoriseAs("test-service");
//...
    private void authoriseAs(String service) {
        Mockito.when(authorisationService.authorise(Mockito.anyString())).thenReturn(service);
    }
```

Reply:
Please use static imports instead of using `Mockito` class name.